devices/fan_1.py

```diff
@@ -10,15 +10,7 @@
 app = Flask(__name__)
 api = Api(app)
 
-light_1 = Light('Lampka-nocna')
-light_2 = Light('Lampka-ścienna')
-light_3 = Light('Oświeglenie-górne')
 fan_1 = Fan('wiatrak-w-salonie')
-fan_2 = Fan('wiatrak-w-Kuchni')
-
-print(type(fan_1.get_method))
-
-
 class Finder(Resource):

```

Comment:
As `Finder` name is general, this class should provide universal returns. In this case - it's only fan_1.